workers.tf

```diff
@@ -73,7 +73,7 @@ resource "aws_security_group_rule" "workers_ingress_cluster" {
   protocol                 = "tcp"
   security_group_id        = "${aws_security_group.workers.id}"
   source_security_group_id = "${local.cluster_security_group_id}"
-  from_port                = 1025
+  from_port                = 0
```

Comment:
Let's make this a variable instead and default it to 1025. Name it something like `worker_sg_ingress_from_port` and provide a suitable description.